app/assets/javascripts/admin.js

```diff
@@ -24,7 +24,8 @@ var assignmentsReady = function(){
 
 }
 
-$(document).ready(assignmentsReady)
+$(document).ready(assignmentsReady);
+$(document).on('page:load', assignmentsReady);
```

Comment:
Is it going to cause issues if this function is run more than once? I.E. when the `$(document).ready` fires, and when the `page:load` fires?